### ISSUE TITLE
Tritinia Scans URL Change

### ISSUE THREAD

**Issue by mostidaniel** [tachiyomiorg/tachiyomi-extensions#14598]
### Source information and language

Tritinia Scans (1.3.27)

### Source new URL

https://tritinia.org

### Other details

The current domain (https://tritinia.com) redirects to a new domain. (https://tritinia.org) Tested in WebView as well as Chrome and Firefox. Currently the extension returns error 405 in the app.

I have looked at the repo branch beforehand and I can confirm that the version for Tritinia available there is also 1.3.27. I have also looked at the issues listing and after using the filter to search Tritinia specifically, I can see that the last issue regarding Tritinia was from 2021 and was already closed.

Additional note:
From what I've read on their Discord server, my understanding is that their original domain was blocked on AdSense by Google. Hence, they decided to change to a new domain. If I recall correctly, the server and content itself isn't any different.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated all installed extensions.
- [X] I have checked if the source URL is not already updated by opening WebView.
- [X] I will fill out all of the requested information in this form.
